### DISCUSSION TITLE
[FEATURE]: New UI

### DISCUSSION THREAD

**Discussion by Bashamega** [rbjks/rbjks.github.io#131] [Ideas]
### Is this a unique feature?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Is your feature request related to a problem/unavailable functionality? Please describe.

Hello:)
I think the design of the website is not good, and it is very basic, and most of the text we can't read.

### Proposed Solution

My solution is to update the website and redesign the website. I was thinking something like this but for light mode
https://templatesjungle.com/downloads/joe-free-figma-personal-website-design-template/

### Do you want to work on this issue?

Yes

### If "yes" to above, please explain how you would technically implement this

I will re-write all the code

**Comment by adityaraute** [comment]
@Bashamega I can't see how this web design fits our website. Could you elaborate or provide more specific examples about what we are lacking at the moment?
